src/js/utils/MarathonConfigUtil.js

```diff
@@ -0,0 +1,73 @@
+import HostUtil from '../utils/HostUtil';
+import Networking from '../constants/Networking';
+
+const serviceAddressKey = 'Service Address';
+
+function buildHostName(id, port) {
+  let hostname = HostUtil.stringToHostname(id);
+  return `${hostname}${Networking.L4LB_ADDRESS}:${port}`;
+}
+
+function defaultCreateLink(contents) {
+  return contents;
+}
+
+var MarathonConfigUtil = {
+
+  getCommandString(container) {
```

Comment:
Let's please add tests to make verify that this util works as expected. Please test especially possible pod/app conflicts.